code/modules/modular_computers/file_system/programs/alarm.dm

```diff
@@ -40,7 +40,7 @@
 	return data
 
 /datum/computer_file/program/alarm_monitor/proc/triggerAlarm(class, area/A, O, obj/source)
-	if(!is_station_level(source.z) && !is_mining_level(source.z))
+	if((!is_station_level(source.z) && !is_mining_level(source.z)) || istype(A, /area/ruin))
```

Comment:
check it against `GLOB.the_station_areas` instead imo

Reply:
didn't even realize that was a thing, sure.